build.sbt

```diff
@@ -57,7 +58,7 @@ lazy val root = (project in file("."))
     libraryDependencies ++= commonDependencies ++ Seq(
       ws,
       filters,
-      "com.gu" %% "play-googleauth" % "0.7.4",
+      "com.gu.play-googleauth" %% "play-v27" % "1.0.7",
```

Comment:
This is the key update. We'll switch over to use the non-deprecated use of this version in a subsequent PR so that the details aren't hidden by upgrade boiler-plate.